server/src/main/java/org/opensearch/search/aggregations/bucket/composite/CompositeAggregator.java

```diff
@@ -166,21 +167,22 @@ public final class CompositeAggregator extends BucketsAggregator {
         this.rawAfterKey = rawAfterKey;
 
         fastFilterContext = new FastFilterRewriteHelper.FastFilterContext(context);
```

Comment:
Any reason for initializing fastFilterContext here? Can it be done after if check along with setting aggregationType?

Reply:
Then fastFilterContext may be null if composite aggregator not rewriteable